target/step13-2020-1/Pages/user.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
hmmm it looks like there are two copies of similar (if not identical) code, why's that the case? i.e.

src/main/webapp/Pages/user.html
src/main/webapp/index.html
src/main/webapp/js/imgHandling.js

target/step13-2020-1/Pages/user.html
target/step13-2020-1/index.html
target/step13-2020-1/js/imgHandling.js